### PR TITLE
new output for iptables and add rest in ret

### PR DESCRIPTION
**Pull request by marcin-usielski**

### CODE REVIEW

File: moler/cmd/unix/iptables.py [hunk 1]
```diff
@@ -65,32 +68,59 @@ def _parse_chain_references(self, line):
...
-        if self._regex_helper.search_compiled(Iptables._re_parse_rest, line):
-            ret = dict()
-            ret["REST"] = self._regex_helper.group("REST")
-            self.current_ret[self.chain]["CHAIN"].append(ret)
-            raise ParsingDone
+            if len(values) >= len(self._key_details):
+                for value, key in zip(values, self._key_details):
+                    ret[key] = value
+                self.current_ret[self.chain]["CHAIN"].append(ret)
+                regex_for_rest = ""
```
Comment (tkrol88) [inline moler/cmd/unix/iptables.py:111]: Could you extraxt below code to private method?
```
                regex_for_rest = ""
                i = 0
                while i < len(self._key_details):
                    regex_for_rest = r"{}\S+\s+".format(regex_for_rest)
                    i += 1
                regex_for_rest = r"{}(?P<REST>\S.*\S|\S+)".format(regex_for_rest)
                re_for_rest = re.compile(regex_for_rest)
```

File: moler/cmd/unix/iptables.py [hunk 2]
```diff
@@ -65,32 +68,59 @@ def _parse_chain_references(self, line):
...
 
     def _parse_headers(self, line):
-        if self._regex_helper.search_compiled(Iptables._re_parse_headers, line):
+        if self.chain and 0 == len(self._key_details):
+            matched = re.findall(r"\s*(\S+)\s*", line)
+            if matched:
+                for header in matched:
+                    self._key_details.append(header.upper())
             raise ParsingDone
+        # if self._regex_helper.search_compiled(Iptables._re_parse_headers, line):
```
Comment (tkrol88) [inline moler/cmd/unix/iptables.py:97]: remove commented code

File: moler/cmd/unix/iptables.py (hunk 1 above)
Comment (marcin-usielski) [reply moler/cmd/unix/iptables.py]: done.

File: moler/cmd/unix/iptables.py (hunk 2 above)
Comment (marcin-usielski) [reply moler/cmd/unix/iptables.py]: done

**Pull request merged**